packages/web3-validator/src/types.ts

```diff
@@ -143,10 +143,7 @@ export type Schema = {
 };
 export interface Validate {
 	(value: Json): boolean;
-	errors?: ValidationError[];
+	errors?: ZodIssueBase[];
```

Comment:
This will be breaking change, as Validation interface is being exported and available to users. For this during release we will need to bump major version of validator package. so:

1. mention this in changelog
2. I think create some validation error wrapper around `ZodIssueBase` , so we are not tied to specific lib (like we were is-my-json-valid.ValidationError)

Reply:
ValidationError wrapper was added